src/selenium/trackPage.js

```diff
@@ -0,0 +1,248 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const BasePage = require('./basePage.js');
+const By = require('selenium-webdriver').By;
+
+const TrackPage = Object.create(BasePage);
+
+// Get number of visible tracks on the page
+TrackPage.getNumTracksVisible = function() {
+  const self = this;
+  const numPromise = new Promise(function(resolve) {
+    self.findAll(By.className('track-filter')).then(function(trackElems) {
+      let counter = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 spaces but found 6.](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)